pyglet/shapes.py

```diff
@@ -719,6 +731,10 @@ def __init__(self, x, y, radius, segments=None, color=(255, 255, 255, 255),
         self._create_vertex_list()
         self._update_vertices()
 
+    def __contains__(self, point):
+        assert len(point) == 2
+        return math.dist((self._x - self._anchor_x, self._y - self._anchor_y), point) < self._radius
```

Comment:
1: You have __contains__ twice
2: You don't actually check if the point is in any of the shapes

Reply:
Can you explain it in detail?

Reply:
I am dumb never mind